routes/game.rb

```diff
@@ -36,6 +40,14 @@ class Api
             game.to_h
           end
 
+          # POST '/api/game/<game_id>/user_settings'
+          r.is 'user_settings' do
+            DB.with_advisory_lock(:action_lock, game.id) do
```

Comment:
i don't think this lock is necessary

Reply:
If two users try to save the notepad at the same time, won't one user lose their content?

Reply:
@tobymao do you want me to remove the lock? I think it might have race conditions if we remove it.